src/07-projects/projects.apib

```diff
@@ -126,6 +126,12 @@ Create a new project.
                         + decision_maker
                         + member
                     + Default: `member`
+        + customer (object, optional)
+            + type (enum[string])
```

Comment:
PP: add `required` to the definition (same for `id`). It is required whenever the customer is provided.